testutils/ptest2/pom.xml

```
@@ -26,7 +26,7 @@ limitations under the License.
   <name>hive-ptest</name>
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <log4j2.version>2.10.0</log4j2.version>
+    <log4j2.version>2.17.0</log4j2.version>
```

```suggestion
    <log4j2.version>2.17.1</log4j2.version>
```
To cover CVE-2021-44832